__tests__/frontend-proxy.ts

```diff
@@ -149,6 +150,20 @@ describe('handleRequest()', () => {
       expect(await getCachedType('/math')).toBe('Page')
     })
 
+    test('api requests are signed', async () => {
+      const mockedFetch = mockFetch({
+        'https://api.serlo.org/': createApiResponse('Page'),
+        'https://frontend.serlo.org/math': '',
+      })
+      global.FRONTEND_PROBABILITY = '1'
+      global.FRONTEND_ALLOWED_TYPES = '["Page"]'
+
+      const request = new Request('https://de.serlo.org/math')
+      await handleRequest(request)
+
+      expect(getApiAuthorization(mockedFetch)).toMatch(/Serlo Service=ey.*/)
+    })
```

Comment:
This test is only needed here because you are depending on the implementation details when doing api requests. Instead, the following would be preferable in my opinion:
- Instead of exposing `signApiRequest`, expose something like `fetchApi` which accepts the GraphQL query and does the request (including sending the token).
- You can then add unit tests for `fetchApi` (e.g. the one you added that checks that it sends the token)
- `frontend-proxy` can then use that function and does not need to know that API requests are signed or how they are handled (e.g. POST request vs. GraphQL client).